### ISSUE TITLE
Please include commit on master branch in 1.0 branch also

### ISSUE THREAD

**Issue by DavidAnderson684** [phpseclib/phpseclib#1922]
Hi,

https://github.com/phpseclib/phpseclib/commit/ba4414c2f3788b64de17e922d0496f6913c05898

Please can this change also be made to the 1.0 branch? I have a user who is getting his logs filled with...

`DEPRECATED: Implicit conversion from float to int loses precision (line 3525, vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php`

... who reports that making this change prevents that.

David

**Comment by terrafrost** [comment]
That change is in the 1.0 branch but I haven't released a new version of 1.0 since I made that change. I can do so in the next few days if that'd be helpful?

For good measure took a look at SSH2.php in https://wordpress.org/plugins/ssh-sftp-updater-support/ and... 
 https://plugins.svn.wordpress.org/ssh-sftp-updater-support/trunk/phpseclib/Net/SSH2.php Line 3525 in that is a new line with the surrounded lines as follows:

```php
        //if ($this->decompress) {
        //    $payload = gzinflate(substr($payload, 2));
        //}

        $this->get_seq_no++;
```
Altho SSH2.php doesn't live in the vendor folder with that..

**Comment by DavidAnderson684** [comment]
@terrafrost Thank you, yes, a new release would be very helpful.
I didn't follow what you were saying about the WordPress plugin? SVN trunk there isn't being used for development. The current version is https://plugins.trac.wordpress.org/browser/ssh-sftp-updater-support/tags/0.8.5/phpseclib/Net/SSH2.php#L3525 .

**Comment by terrafrost** [comment]
A new version of 1.0 has been released! (1.0.21)
